### DISCUSSION TITLE
Did you want mark an App?

### DISCUSSION THREAD

**Discussion by leonardwoo** [emqx/MQTTX#937] [Ideas]
Did you want made an App for developer/debug on Android and iOS？

**Comment by ysfscream** [comment]
> Did you want made an App for developer/debug on Android and iOS？

OMG! sorry for my late reply. There are no plans to do this yet, but we will first go live with a web-based version of the app to help users use it faster. But I actually have a library of private projects and would like to implement an app using RN or Flutter, but I can't do that due to resource constraints.

https://github.com/ysfscream/MQTTX-Mobile

**Reply by leonardwoo** [reply]
請一定要用Flutter，尤其iOS上還沒有很好用的MQTT客戶端。

**Reply by ysfscream** [reply]
OK, I will consider this. But I have a question, is there really a need to use the MQTT test client on a mobile phone?

**Reply by leonardwoo** [reply]
Maybe? There may not be many users, but it is still very comfortable to use any device at your fingertips.
Especially now that the hell knows who will work with what weird device.

**Reply by ysfscream** [reply]
Lol! Indeed, there does seem to be a need to extend MQTT X to other platforms.
